src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -125,13 +125,13 @@ public void run(RunConfiguration config) throws AppEngineException {
   public void stop(StopConfiguration configuration) throws AppEngineException {
     Preconditions.checkNotNull(configuration);
     HttpURLConnection connection = null;
+    String adminHost =
+        configuration.getAdminHost() != null ? configuration.getAdminHost() : DEFAULT_HOST;
```

Comment:
At least the port param is valid in the sense that it is the only way to shutdown the server if it starts on a port other than 8080.